config.json

```diff
@@ -231,6 +231,14 @@
         "prerequisites": [],
         "difficulty": 3
       },
+      {
+        "slug": "rectangles",
+        "name": "Rectangles",
+        "uuid": "2eb30b47-a1e0-44e5-bb22-6085a938ddb5",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
+      },
```

Comment:
Could you update the difficulty and ordering?

Reply:
Oh, good catch, thanks Isaac. 👍